website/api/zkvm/optimization.md

```diff
@@ -511,14 +515,14 @@ below.
 [example-waldo]: https://github.com/risc0/risc0/tree/main/examples/waldo
 [flamegraph]: https://www.brendangregg.com/FlameGraphs/cpuflamegraphs.html
 [golang-install]: https://go.dev/doc/install
-[hibernates]: https://en.wikipedia.org/wiki/Hibernation_(computing)
+[hibernates]: https://en.wikipedia.org/wiki/Hibernation_\(computing\)
```

Comment:
slashes here are no good

Reply:
will take a look through these and see what rule caused them.. thanks Paul!

Reply:
i was suspicious about the fix but indeed https://en.wikipedia.org/wiki/Hibernation_%28computing%29 is a valid alternative to https://en.wikipedia.org/wiki/Hibernation_(computing)